app/code/pagetype/HomePage.php

```diff
@@ -1,11 +1,45 @@
 <?php
 
-namespace App\PageType;
+namespace App\Pagetype;
```

Comment:
I just noticed you added a bunch of code related to HomePage, ideally you would want to create a separate branch for this and keep the footer branch code separate :)

Reply:
Dont forget to remove this and include it in a separate branch :)